functional/security/Crypto/playlist.xml

```diff
@@ -35,6 +35,14 @@
 				<comment>https://github.com/adoptium/aqa-tests/issues/4336</comment>
 				<platform>.*solaris</platform>
 			</disable>
+			<disable>
+				<comment>
+					This target is for those who would like to run upstream fully supported way of how to use this suite.
+					This launcher runs in GH, can enable custom jtreg.jar with upstreamed but unreleased stdout fixes,
+					can correctly enable/disable/setup kerberos tests and works in any CWD.
```

Comment:
This comment is inappropriate, please fix it with the suggested sentence instead of using loaded terms and phrases.  For the record, it is not only for the issue on s390x that others do not want to use run.sh, it is precisely because it has a bunch of extra behaviour that is unneeded and unwanted by those using AQAvit Jenkins pipelines.

Reply:
Sure, I will try to elaborate on it. However there is no single word lying. Maybe except  ` ancient s390x platforms` . Although the s390x was the only platform I was able to get close reproduce the issue. On MACs it was working fine. 

I can not agree that ability to enable the kerberos based thest is unwanted. The only disputable part was tarring of  jtr.xml files, where I offered that I will add support for both.
Similarly I think that any possible user should be warned that `run.sh` is what is used in git hub actions.

As I stated before, I would much more likely create the original target with the original launcher to be usefull for anybody (and I committed myself to it) then do any forking. But it have been already decided.

Reply:
Fact-based:
`<comment>This target is for those using GH actions or the Jenkins jtreg-plugin.  The run.sh script provides extra functionality, tarring of results, downloading of bespoke jtreg.jar from a particular untagged commit SHA that is not yet upstreamed to a release version of jtreg that is used to set up some kerberos tests.</comment>`